src/components/NoticeOfWithdrawal/PlanOfArrangement.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div id="withdrawal-poa">
+    <v-row no-gutters>
+      <v-col
+        cols="12"
+        sm="3"
+        class="pr-4"
+      >
+        <label id="poa-label">Plan of Arrangement</label>
+      </v-col>
+      <v-col
+        cols="12"
+        :sm="9"
+      >
+        <v-checkbox
+          id="plan-of-arrangement-checkbox"
+          v-model="planOfArrangement"
+          class="mt-0 pt-0"
+          hide-details
+          label="The record to be withdrawn was filed as part of an arrangement."
+        />
+      </v-col>
+    </v-row>
+    <v-row
+      no-gutters
+      class="pt-4"
+    >
+      <v-col
+        cols="12"
+        sm="3"
+        class="pr-4"
+      />
+      <v-col
+        cols="12"
+        :sm="9"
+      >
+        <v-checkbox
+          id="come-into-effect-checkbox"
+          v-model="comeIntoEffect"
+          class="mt-0 pt-0"
+          hide-details
+          label="One or more of the provisions of the arrangement have come into effect."
+          :disabled="!planOfArrangement"
+        />
+      </v-col>
+    </v-row>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Emit, Prop, Watch } from 'vue-property-decorator'
+
+  @Component({})
+export default class PlanOfArrangement extends Vue {
+    /** Prompt the validations. Used for global validations. */
+    @Prop({ default: false }) readonly autoValidation!: boolean
+
+    /** Draft plan of arrangement. */
+    @Prop({ default: false }) readonly hasDraftPlanOfArrangement!: boolean
+
+    /** Draft come into effect. */
+    @Prop({ default: false }) readonly hasDraftComeIntoEffect!: boolean
+
+    /** Prompt Error. */
+    @Prop({ default: false }) readonly invalidSection!: boolean
+
+    // Local properties
+    private planOfArrangement = false
+    private comeIntoEffect = false
+    private valid = false
+
+    /** Called when component is mounted. */
+    mounted (): void {
+      // Set default draft values if they exist
+      if (this.hasDraftPlanOfArrangement) this.planOfArrangement = this.hasDraftPlanOfArrangement
+      if (this.hasDraftComeIntoEffect) this.comeIntoEffect = this.hasDraftComeIntoEffect
```

Comment:
This won't update the local properties if the props ever change. Could that ever happen here (eg, draft is loaded after page is displayed)?

As an alternative, you could "watch" the props and use the `{ immediate: true }` option.

Reply:
Yes, sorry, I have not completed the POA section. I thought Jacqueline will remove it from the design. Just confirmed with her we will keep this section for now. I will work on this part

Reply:
I looked at other components, I think it won't happen: CourtOrderPoa.vue

Reply:
So this component isn't rendered (mounted) until a draft filing is resumed (if there is one)?

Reply:
I see you're protected by the `dataLoaded` boolean.

Therefore, as long as the props don't change for another reason, doing it this way, once, in mounted(), is OK.

Reply:
Arwen, please resolve this conversation after you've read it. Thx.